src/permission-set.js

```diff
@@ -410,18 +410,23 @@ class PermissionSet {
       debug('Individual access granted for ' + resourceUrl)
       return Promise.resolve(true)
     }
-    // If there are no group authorizations, no need to proceed
-    if (!this.hasGroups()) {
-      debug('No groups authorizations exist')
-      return Promise.resolve(false)
+
+    if (this.hasGroups()) {
+      // Lastly, load the remote group listings, and check for group auth
+      debug('Check groups authorizations')
+
+      return this.loadGroups(options)
+	    .then(() => {
+		return this.checkGroupAccess(resourceUrl, agentId, accessMode, options)
+	    })
     }
-    // Lastly, load the remote group listings, and check for group auth
-    debug('Check groups authorizations')
 
-    return this.loadGroups(options)
-      .then(() => {
-        return this.checkGroupAccess(resourceUrl, agentId, accessMode, options)
-      })
+    // Then, access will not be granted
+    debug('Can agent ' + agentId + ' ' + accessMode + ' ' + resourceUrl + '?')
+    if (this.origin) {
+      debug('Can origin ' + this.origin + ' ' + accessMode + ' ' + resourceUrl + '?')
+    }
+    return Promise.resolve(false);
```

Comment:
So the answer is always no? Would it be better to have statements rather than questions?

Reply:
It might be. I took this wording from @timbl's text. Perhaps "Check whether agentID should have ..."? or something like that?

Reply:
Let's cross-check with @timbl, but these existing log lines capture more information succinctly (i.e., they also provide the answers):

```
  solid:ACL Permissions on https://robin.kjernsmo.net:8444/favicon.ico for (none): read +3ms
  solid:ACL Permissions on https://robin.kjernsmo.net:8444/favicon.ico for public: read +0ms
```